src/UseCaseContext.ts

```diff
@@ -16,18 +16,16 @@ export default class UseCaseContext {
     dispatcher: UseCase;
 
     /**
-     * @param   [dispatcher]    dispatcher is Dispatcher or parent UseCase.
-     * @public
+     * @param   dispatcher
+     *  The parent UseCase.
      */
     constructor(dispatcher: UseCase) {
```

Comment:
@azu I forget to ask you about the following question to you.

1. In the document, this `dispatcher` arguments is explained as "parent use case", and this value is passed as `UseCaseExecutorArgs.parent` in `UseCaseExecutorArgs.useCase()`.
2. in jsdoc, this `dispatcher` is marked as optional arguments.

By these things, I worry whether we should make it an optional argument by adding `null` default value like:

```typescript
constructor(dispatcher: UseCase = null);
```

How about do you think?